.github/actions/deploy-backend/action.yml

```diff
@@ -333,7 +332,10 @@ runs:
 
     - name: Validate function app checksum
       if: inputs.checksum-validation == 'true'
+
       uses: JosiahSiegel/checksum-validate-action@ebdf8c12c00912d18de93c483b935d51582f9236
+      ## emvaldes@hotmail.com (Replace) uses: ./.github/actions/checksum-validate-action
```

Comment:
corp email please

Reply:
I am using this as the contributor's email as it's the one for the account that is registered in GitHub.
Nevertheless, if required I will change it.

Reply:
I have updated the email address to reflect the Aquia corporate email.